test/integration.coffee

```diff
@@ -0,0 +1,151 @@
+expect            = require("chai").expect
+TransloaditClient = require "../src/TransloaditClient"
+request           = require "request"
+stream            = require "stream"
+localtunnel       = require "localtunnel"
+http              = require "http"
+url               = require "url"
+
+authKey    = process.env.TRANSLOADIT_KEY
+authSecret = process.env.TRANSLOADIT_SECRET
+unless authKey? && authSecret?
+  msg  = "specify envrionment variables TRANSLOADIT_KEY and TRANSLOADIT_SECRET"
+  msg += " to enable integration tests."
+  console.warn msg
+  return # Terminates module execution without existing the test process
```

Comment:
Great to see the integration tests added! And on a less important note, I noticed a little typo here in `existing`